tests/target/control-brace-style-always-next-line.rs

```diff
@@ -7,24 +7,28 @@ fn main() {
         let bar = ();
     }
 
+
```

Comment:
I am adding blank lines to this and subsequent targets to match the number of blank lines in the corresponding source.